core/app/models/spree/line_item.rb

```diff
@@ -57,6 +57,13 @@ def amount
     def discounted_amount
       amount + promo_total
     end
+    deprecate discounted_amount: :final_amount_without_additional_tax, deprecator: Spree::Deprecation
+
+    # @return [BigDecimal] the amount of this item, taking into consideration
+    #   all non-tax adjustments.
+    def final_amount_without_additional_tax
```

Comment:
i'd just call this `final_amount_without_tax` - a bit confused by why it's called `without_additional_tax` - but maybe that's just me.

Reply:
@alexstoick yeah perhaps I should just call it that.  I've mentioned that on the upstream Solidus PR. Let's just leave it here see what comes out upstream?